prepare_cyrus.hs

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env stack
+-- stack runghc --resolver lts-8.14 --package turtle --install-ghc
```

Comment:
Could you indicate me how should I use this?

(Comments on top of the file, README, etc...)